### DISCUSSION TITLE
Welcome to Habitica_ToDoOvers Discussions!

### DISCUSSION THREAD

**Discussion by Kirska** [Kirska/Habitica_ToDoOvers#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by changpowpow** [comment]
I was wondering if it was possible to create To Do Overs with subtasks.

**Reply by Kirska** [reply]
This is on the to-do list https://github.com/Kirska/Habitica_ToDoOvers/issues/5

**Reply by changpowpow** [reply]
Thank you!

**Comment by evilercalls** [comment]
Hi!  I was wondering whether there was a way that this script could (or was supposed to?) run at midnight local?  When I was still in the states, the next morning (at least before 5am MT) the new tasks would appear on my to-do list.  Now I'm in Europe in CET/CEST, and the new tasks do not appear until after 1pm local, if not later.  Thank you very much!
